includes/Reconciliation/ReconciliationItem.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace MediaWiki\Extension\WikibaseReconcileEdit\Reconciliation;
+
+use Wikibase\DataModel\Entity\Item;
+
+/**
+ * An item found by the {@link ReconciliationService}, together with its revision ID.
+ *
+ * This class contains an item as it was found in the database,
+ * or a newly created item if no existing item was found,
+ * in which case the revision ID is false.
+ * It should not be confused with {@link ReconciledItem},
+ * which includes modifications made to the item based on the input.
+ *
+ * @license GPL-2.0-or-later
+ */
+class ReconciliationItem {
```

Comment:
How do you feel about naming this `ReconciliationServiceItem` ?

Even though we have the comment not to confuse the two i think making the difference a bit more obvious will make this less likely to be confused?

Reply:
Sounds okay to me. Or maybe even `ReconciliationServiceResult`, without “item”?

Reply:
Hm, on the other hand the term “item” is used a lot for these results (e.g. `EditRequest::otherItems()`), and changing those to “result” would probably make it less clear overall.

Let’s go with `ReconciliationServiceItem`.